Formula/offlineimap.rb

```diff
@@ -68,7 +82,7 @@ def caveats
   end
 
   service do
-    run [opt_bin/"offlinemap", "-q", "-u", "basic"]
+    run [bin/"offlinemap", "-q", "-u", "basic"]
```

Comment:
anything in `bin` is symlinked from `opt_bin`, I doubt this will help

Reply:
ah, good to know.

Why would the `brew test offlineimap` succeed then?

Reply:
Are there docs on these special locations?

Reply:
This is where brew obviously puts it per default and it's fine:
```
bash-5.1$ type offlineimap
offlineimap is /usr/local/bin/offlineimap
bash-5.1$ offlineimap --version
8.0.0
```

Reply:
I just found `"#{bin}/binary"` in uftp.rb. Is this how I am supposed to do it?

Reply:
What does `ls -al /usr/local/bin/offlineimap` say?

Reply:
@SMillerDev  it's a symlink
```
bash-5.1$ ls -alh /usr/local/bin/offlineimap
lrwxr-xr-x  1 florian  staff    45B Dec  6 12:55 /usr/local/bin/offlineimap -> ../Cellar/offlineimap/8.0.0_1/bin/offlineimap
```

Reply:
```
$ ls -alh /usr/local/Cellar/offlineimap/8.0.0_1/bin/
total 4.0K
drwxr-xr-x  3 florian admin  96 Dec  6 12:55 .
drwxr-xr-x 13 florian admin 416 Dec  6 12:55 ..
-r-xr-xr-x  1 florian admin 180 Dec  6 12:55 offlineimap

$ less /usr/local/Cellar/offlineimap/8.0.0_1/bin/offlineimap
#!/bin/bash
PYTHONPATH="/usr/local/Cellar/offlineimap/8.0.0_1/libexec/vendor/lib/python3.9/site-packages" exec "/usr/local/Cellar/offlineimap/8.0.0_1/libexec/offlineimap.py"  "$@"

$ less /usr/local/Cellar/offlineimap/8.0.0_1/libexec/offlineimap.py
#!/usr/bin/env python3
# Startup from system-wide installation
...
```

Reply:
@SMillerDev  there was a typo :(. Took way too long to spot. offlinemap instead of offlineimap /usr/local/bin/offlinemap

Reply:
It was in master even. No wonder we did not spot it
```
$ git blame Formula/offlineimap.rb | grep offlinemap
Blaming lines: 100% (82/82), done.
4fe8c04ffe2f Formula/offlineimap.rb  (Sean Molenaar        2021-11-06 13:45:26 +0100 71)     run [opt_bin/"offlinemap", "-q", "-u", "basic"]
```